danger_python/models.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass
+from typing import List
+
+from dataclasses_json import dataclass_json
+
+
+@dataclass_json
+@dataclass
+class GitDSL:
+    modified_files: List[str]
+    created_files: List[str]
+    deleted_files: List[str]
+
+
+@dataclass_json
+@dataclass
+class GithubPullRequestDSL:
+    title: str
+
+
+@dataclass_json
+@dataclass
+class GithubDSL:
+    pr: GithubPullRequestDSL
+
+
+@dataclass_json
+@dataclass
+class DangerDSL:
+    git: GitDSL
+    github: GithubDSL
```

Comment:
You _could_ generate this file from the [JSON schema](https://github.com/danger/danger-js/blob/master/source/danger-incoming-process-schema.json) - which is how I did it [for rust](https://github.com/danger/rust/blob/master/src/bin/update_types.rs)

Reply:
In Swift we didn't because we wanted the DSL to feel more Swifty, which is a conscious choice that you can also make

Reply:
That's a great suggestion, I'll take a look to see what are the options to do it. Since we are looking at 2,000 lines already, how about I include it in the next batch of changes?

Reply:
Yep, no need to try get everything at once